test/cpp/interop/BUILD

```diff
@@ -248,6 +249,7 @@ grpc_cc_binary(
     ],
     deps = [
         "//:grpc++",
+        "//:grpcpp_channelz",
         "//src/proto/grpc/testing:empty_proto",
```

Comment:
strange, the auto generated file did a replace instead of add; i wonder if you could remove "//:grpc++" from the BUILD file ?

Reply:
I can, since grpcpp_channelz depends on grpc++ but that seems a bit brittle, given that grpc++ is the target that we expect applications to use normally.